examples/MNIST/output.sh

```diff
@@ -0,0 +1,78 @@
+[
```

Comment:
This isn't really a `.sh` file, it's a `.json`

Reply:
Oh. okay!